### ISSUE TITLE
Firewall Log is broken

### ISSUE THREAD

**Issue by R4toLoko** [opnsense/core#6929]
**Describe the bug**

Firewall Live log no longer displays the filters and the templates I had saved.
I have just updated it with the latest update so this is a bug introduced with the update:

OPNsense 23.7.6-amd64
FreeBSD 13.2-RELEASE-p3
OpenSSL 1.1.1w 11 Sep 2023

**To Reproduce**

1. Go to 'Firewall'
2. Click on 'Log Files'
3. Click on  'Live view'

**Expected behavior**

All the filters options and the templates I have created and was working just fine should be there but everything disappeared.
You can see the gif below, it is logging things just fine and the time is right too.

**Screenshots**

![Peek 2023-10-12 15-54](https://github.com/opnsense/core/assets/35049512/6c3ddcef-cf5b-45e7-b4d8-1b9ed52dec68)

**Relevant log files**

Gif above, data is being logged just fine, general and plain view looks correct.

**Additional context**

N/A other than it has been running fine for yearS with S and this is the first bug I am aware of.
I have been exploring the firewall and its logs a lot to block IoT, everything was perfectly fine prior to the update.

**Environment**

Software version used and hardware type if relevant, e.g.:

OPNsense 23.7.6-amd64
FreeBSD 13.2-RELEASE-p3
OpenSSL 1.1.1w 11 Sep 2023

Intel(R) Core(TM) i5-4210U CPU ( Proxmox)

**Comment by fichtner** [comment]
Cicada theme I suppose?

**Comment by R4toLoko** [comment]
> Cicada theme I suppose?

Yes and actually, it is not just the firewall, everything with drop down menu disappeared, they are all blank like that gif

I took a snapshot (Proxmox) before the update and this isn't blocking me yet so hopefully, this is a quick fix.

Let me change the title.

**Comment by R4toLoko** [comment]
Thank you for the reply.
